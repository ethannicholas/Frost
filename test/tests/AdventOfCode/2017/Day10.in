63,144,180,149,1,255,167,84,125,65,188,0,2,254,229,24