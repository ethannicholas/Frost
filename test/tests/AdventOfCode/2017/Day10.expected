4480
c500ffe015c83b60fad2e4b7d59dabc4
